mypy/build.py

```diff
@@ -375,11 +379,21 @@ def plugin_error(message: str) -> None:
                 '(in {})'.format(plugin_path))
         try:
             custom_plugins.append(plugin_type(options))
+            # We record _both_ hash and the version to detect more
```

Comment:
This function is quite long. It would be better if this would be moved to a helper method to avoid making it even longer.